tools/esdlcmd/esdlcmd_shell.cpp

```diff
@@ -180,7 +180,7 @@ bool EsdlCMDShell::parseCommandLineOptions(ArgvIterator &iter)
 void EsdlCMDShell::usage()
 {
     fprintf(stdout,"\nUsage:\n"
-        "    esdl [--version] <command> [<args>]\n\n"
```

Comment:
I did add usage for --build in the common command usage. What I'm trying to do is to avoid overloading the meaning of --version, as although we're removing it from publish and java, it's still being used for specifying esdl version in xsd and wsdl commands.